### PR TITLE
remove BCP14 language from introduction

### LINKED ISSUE #38: SID: MK issues, now split up
Section 1:

It's a bit weird to have requirements language in an Introduction.

Section 7, generally:

Thanks for providing advice for the Designated Experts for each of the created registries.  This is a not-exactly-uncommon oversight.

### PR DESCRIPTION
**Pull request by mcr**
close the comment on Section 1 in #38

### CODE REVIEW

File: draft-ietf-core-sid.md [hunk 1]
```diff
@@ -147,16 +147,16 @@ guarantee their uniqueness. SIDs are registered in blocks called "SID ranges".
...
 
 {{sid-lifecycle}} provides more details about the registration process of YANG
 modules and associated SIDs. To enable the implementation of this registry,
 {{sid-file-format}} defines a standard file format used to store and publish
 SIDs.
 
-IETF managed YANG modules which need to allocate SIDs, MUST use the IANA mechanism specified in this document.
-For YANG modules created by other parties, the use of IANA allocation mechanisms via use of Mega-Ranges (see {{mega-range-registry}}) is RECOMMENDED.
+IETF managed YANG modules which need to allocate SIDs use the IANA mechanism specified in this document.
+For YANG modules created by other parties use the IANA allocation mechanisms via Mega-Ranges (see {{mega-range-registry}}).
```
Comment (cabo) [inline draft-ietf-core-sid.md:159]: Imperative?

File: draft-ietf-core-sid.md [hunk 2]
```diff
@@ -147,16 +147,16 @@ guarantee their uniqueness. SIDs are registered in blocks called "SID ranges".
 SIDs are assigned permanently.
 Items introduced by a new revision of a YANG
 module are added to the list of SIDs already assigned.
-Assignment of SIDs to YANG items SHOULD be automated.
-For more details how this can be achieved, and when manual interventions may be appropriate, see {{sid-auto-generation}}.
+Assignment of SIDs to YANG items can be automated using the methods from {{sid-auto-generation}}.
```
Comment (cabo) [inline draft-ietf-core-sid.md:150]: Hmm, we already showed the previous change to the IESG and a DISCUSS got cleared, IIRC.  Why are we changing this again?

File: draft-ietf-core-sid.md (hunk 1 above)
Comment (mcr) [reply draft-ietf-core-sid.md:159]: It seemed a good way to remove the BCP14 language from there.
If you don't like it, I don't think anyone is strongly attached to this change.

**Comment by cabo** [comment]
OK, I think I have preserved the intent of the original DISCUSS resolution now.
I believe this can now be merged.

**Pull request merged**